api/src/main/resources/workbench-api.yaml

```diff
@@ -6027,6 +6027,10 @@ definitions:
         description: Prefix of the research VM(s) that generated egress, including
           the user ID, e.g. all-of-us-260.
         type: string
+      workspace_namespace:
```

Comment:
This is a bit misleading - the incoming JSON event value does not actually include this, it gets tacked on later by the controller.

I would either remove it and pass that separately alongside the EgressEvent on the server side (this is probably the most consistent), or else leave it (this is more convenient) and put a very clear comment here about why this is here and when it gets populated - and the comment should not say "always present".

Reply:
Removed that and set it in each methods.(it does make the code looks not great, but that is my another unrelated question since I read this yaml file about using Swagger as HTTP pojo as well as service's internal pojo. I do have opinion on we should separate them, and find solutions for 2nd use case, e.g. `autoValue`) 
Initially I make that a field because I thought that `audit.kt` file was a separate service that is communicated with RW over HTTP. But I just make that change to also load workspace from googleProject. PTAL